components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -1110,22 +1102,13 @@ private String getNextStringInLexicalOrder(String str) {
         return new String(bytes);
     }
 
-    private OMElement queryDatabase(String columnFamily,
-                                    QueryServiceStub.CompositeIndex[] compositeIndex)
-            throws APIMgtUsageQueryServiceClientException {
+    private OMElement queryDatabase(String columnFamily) throws APIMgtUsageQueryServiceClientException {
```

Comment:
To me this is a bad method name. Add more context into what this method does into its name. Please fix.